bin/storm.py

```diff
@@ -541,6 +541,13 @@ def initialize_upload_credentials_subcommand(subparsers):
         help="""name of the owner of the topology (security precaution)"""
     )
 
+    # If set, this flag will become true meaning that user expects non-empty creds to be uploaded.
+    # Command exits with non-zero code if uploaded creds collection is empty.
+    sub_parser.add_argument(
+        "-e", "--exception-when-empty", action='store_true',
+        help="""whether to throw exception if there are no credentials uploaded, default to be false"""
```

Comment:
It means true when this config is specified, otherwise it's false. 

So more precisely, "If specified, throws an exception if no credentials are uploaded"

Reply:
Fixed